nix/cells/automation/library/plutus-benchmark-runner.sh

```diff
@@ -81,9 +68,11 @@ EOF
 echo -e "</details>"
 } > bench-compare-result.log
 
-nix-shell -p jq --run "jq -Rs '.' bench-compare-result.log >bench-compare.json"
-# TODO(std) check that this works
-# nix develop --command "jq -Rs '.' bench-compare-result.log >bench-compare.json"
-
-echo "[ci-plutus-benchmark]: Posting results to GitHub ..."
-curl -s -H "Authorization: token $(</run/keys/buildkite-github-token)" -X POST -d "{\"body\": $(<bench-compare.json)}" "https://api.github.com/repos/input-output-hk/plutus/issues/${PR_NUMBER}/comments"
+jq -Rs '.' bench-compare-result.log >bench-compare.json
+if [ -v GITHUB_TOKEN ] ; then
+   echo "[ci-plutus-benchmark]: Posting results to GitHub ..."
+   # TODO(std) we'll need to change this
+   curl -s -H "Authorization: token $(<$GITHUB_TOKEN)" -X POST -d "{\"body\": $(<bench-compare.json)}" "https://api.github.com/repos/input-output-hk/plutus/issues/${PR_NUMBER}/comments"
+else
+   echo "[ci-plutus-benchmark]: GITHUB_TOKEN not set, not posting results to GitHub"
```

Comment:
This should probably be a failure. Are we going to get a commit status or some way of knowing if the job failed?

Reply:
I did it like this because you wanted the script to be able to be run locally, I assume you wouldn't want to have to get a github token if your just getting info on the command line.

Reply:
That's a great point. Maybe I'm just talking nonsense, I'm no longer sure what should be true of this script :grimacing:  I guess leave it like this for now.